lang/py/avro/schema.py

```diff
@@ -774,6 +843,13 @@ def to_json(self, names=None):
         to_dump['values'] = self.get_prop('values').to_json(names)
         return to_dump
 
+    def to_canonical_json(self, names):
```

Comment:
Should names be optional?

Reply:
Yes. Addressed in eacfdd6a0cd6de9b4e15701d08d9e994f09be310